main/src/ca/uwaterloo/flix/language/phase/Synthesize.scala

```diff
@@ -821,6 +839,10 @@ object Synthesize extends Phase[Root, Root] {
           val method = classOf[java.lang.Object].getMethod("toString")
           Expression.NativeMethod(method, List(exp0), Type.Str, ast.Eff.Pure, sl)
 
+        case Type.Channel =>
```

Comment:
A question for the report, but probably outside the scope of the implementation, is whether channels should support equality and if so, what would it mean?